src/views/Main/Main.js

```diff
@@ -0,0 +1,63 @@
+import { fetchByType, fetchPokemon, fetchType } from '../../services/pokemon';
+import { usePokeContex } from '../../context/PokeContext';
+import Search from '../../components/Search';
+
+import './Main.css';
+
+import React, { useEffect, useState } from 'react';
+import PokeList from '../../components/PokeList';
+
+export default function Main() {
+  // const { selectedType, pokemon, setPokemon, search, order, setTypes } = usePokeContex();
+  const [pokemon, setPokemon] = useState([]);
+  const [selectedType, setSelectedType] = useState('All');
+  const [search, setSearch] = useState('');
+  const [order, setOrder] = useState('asc');
+  const [types, setTypes] = useState([]);
+  const [load, setLoad] = useState(false);
+
```

Comment:
Don't forget to add in a loading state for when the data loads and for when someone searches for a pokemon.